features/verblijfplaatshistorie.feature

```diff
@@ -1,182 +1,626 @@
+    | 999992338           | 58        |                            | 19960000                       | 19960000                |                     |            | 5010 |           |                        |                       |
+    | 999992338           | 8         | 20010508                   |                                | 20010508                | Leyweg              | 61         | 6030 |           |                        |                       |
+    | 999993483           | 8         | 20120305                   |                                | 20120305                | Zonegge             | 27         | 6030 |           |                        |                       |
+    | 999993483           | 58        | 00000000                   |                                | 00000000                | Leyweg              | 61         | 6030 |           |                        |                       |
+    | 999990378           | 8         | 20150601                   |                                |                         | Rietzangerstraat    | 20         | 6030 |           |                        |                       |
+    | 999990378           | 58        | 19881118                   |                                |                         | Voorhofdreef        | 30         | 6030 | 580000    | 19881120               |                       |
+    | 999990263           | 8         | 20150601                   |                                |                         | Rapheëlstraat       | 1          | 6030 |           |                        | 20120305              |
+    | 999990263           | 58        | 19890501                   |                                |                         | Oudegracht          | 380        | 6030 | 580000    |                        | 19920102              |
+    | 999990451           | 8         | 20130301                   |                                | 20130302                | Chilidreef          | 14         | 6030 |           |                        |                       |
+    | 999990451           | 58        | 19890000                   |                                | 20120302                | Dennestraat         | 35         | 6030 |           |                        |                       |
+    | 999990762           | 8         | 20140203                   |                                | 20140203                | Jan Provostlaan     | 172        | 6030 |           |                        |                       |
+    | 999990762           | 58        | 19891000                   |                                | 19930203                | Cypresstraat        | 14         | 6030 |           |                        |                       |
+
+
+Rule: Bij het zoeken is de peildatum verplicht of zijn de datumVan en datumTotEnMet verplicht.
```

Comment:
is het niet handiger dit te groeperen per zoektype?

- Bij raadplegen met peildatum zijn peildatum, burgerservicenummer en fields verplicht
- Bij raadplegen met periode zijn datumVan, datumTot, burgerservicenummer en fields verplicht

maar je kan het ook indelen zoals in de zoek-met-... features. Dus voor elk zoektype een feature